lint/parenthesis-checker.go

```diff
@@ -0,0 +1,46 @@
+package lint
+
+import (
+	"fmt"
+	"go/ast"
+)
+
+// ParenthesisChecker ...
```

Comment:
Don't be too lazy.
Either write at least proper sentence, or drop it completely and let linter rage continue (someone will fix that later).
You're only making it harder to refactor code later (fooling around linter).

Reply:
It's actually very common bad practice to use `// Foo ...` or something like that.
We could add a checker for that.

Reply:
thank you, i have created an issue for that #24